src/components/AppBar/AppBar.tsx

```diff
@@ -0,0 +1,130 @@
+"use client"
```

Comment:
Given the number of Svelte changes, I'd say let's make those first. Then that can inform the component "api" for React.